src/main/kotlin/org/malachite/estella/interview/api/InterviewExceptionHandler.kt

```diff
@@ -0,0 +1,27 @@
+package org.malachite.estella.interview.api
+
+import org.malachite.estella.commons.Message
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.commons.UnauthenticatedException
+import org.malachite.estella.interview.domain.InterviewNotFoundException
+import org.malachite.estella.interview.domain.InvalidUUIDException
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
+class InterviewExceptionHandler {
+
+    @ExceptionHandler(InterviewNotFoundException::class)
+    fun handleNoSuchElementException(ex: InterviewNotFoundException): ResponseEntity<Any> =
+            ResponseEntity(Message("We couldn't find this interview"), HttpStatus.NOT_FOUND)
+
+    @ExceptionHandler(UnauthenticatedException::class)
```

Comment:
Wydaje mi się, że taki handler już gdzieś mamy, więc ten jest nie potrzebny, ale to jest do sprawdzenia